code/game/mob/living/carbon/human/human_defense.dm

```diff
@@ -758,19 +775,19 @@ bullet_act
 		if (istype(user, /mob/living/human))
 			var/mob/living/human/HH = user
 			if (prob(6*HH.getStatCoeff("dexterity")))
-				visible_message("<span class='danger'>[src] has been knocked down!</span>")
+				user.visible_message("<span class='danger'>[src] has been knocked down!</span>", "<span class='danger'><h4>You are knocked down!</h4></span>")
 				Weaken(2)
 		else
 			if (prob(6))
-				visible_message("<span class='danger'>[src] has been knocked down!</span>")
+				user.visible_message("<span class='danger'>[src] has been knocked down!</span>", "<span class='danger'><h4>You are knocked down!</h4></span>")
 				Weaken(2)
 	instadeath_check()
 
-/mob/living/human/proc/attack_joint(var/obj/item/organ/external/organ, var/obj/item/W, var/blocked)
+/mob/living/human/proc/attack_joint(var/obj/item/organ/external/organ, var/obj/item/W, mob/living/user, var/blocked)
```

Comment:
mob/living/user isn't being sent through., you're trying to call to something that does not exist